generative_data_prep/utils/utils.py

```diff
@@ -16,13 +16,14 @@
 import argparse
 from subprocess import PIPE, run  # nosec
 
-from transformers import GPT2Tokenizer
+from transformers import GPT2Config, GPT2Tokenizer
 
 from .arg_configs import PackingConfig
 from .constants import BoundaryType
 
 GPT2_KEY = "gpt2"
 TOKENIZER_CLASSES = {GPT2_KEY: GPT2Tokenizer}
+TOKENIZER_CONFIG = {GPT2_KEY: GPT2Config}
```

Comment:
Can we add an assertion or create this data structure in a way that we ensure that both tokenizer classes and tokenizer configs have the same keys? 

Just so if someone updates it they update both